public/js/main.js

```diff
@@ -63,6 +63,24 @@ $(function() {
 	});
 
 	$('.bio-modal').on('shown.bs.modal', function () {
-  		$(this).find('textarea').focus();
-  	});
+		$(this).find('textarea').focus();
+	});
+
+	$('[data-clipboard]').each(function (i, element) {
+		$(element).tooltip({
+			trigger: 'manual',
+			placement: 'bottom',
+			title: 'Copied!'
+		});
+
+		var client = new ZeroClipboard(element);
+		client.on( "ready", function( readyEvent ) {
+			client.on( "aftercopy", function( event ) {
```

Comment:
Is this `aftercopy` bound inside of `ready` because that's when you have to bind things? It looks weird to nest one listener inside another, but maybe you have to nest everything inside `ready`?